python/lsst/ts/observatory/control/remote_group.py

```diff
@@ -61,6 +61,64 @@ def __iter__(self):
         )
 
 
+class UseCase:
+    """Use case definition for RemoteGroup.
+
+    Parameters
+    ----------
+    components : `str`
+        Name of the components required for this use case. Names must follow
+        the same format as that of the `components` attribute in `RemoteGroup`,
+        which is the name of the CSC in lowercase replacing the ":" by "_",
+        e.g. Hexapod:1 is hexapod_1 and MTDomeTrajectory is mtdometrajectory.
```

Comment:
Are you sure you want this convention? The `component` argument is usually the other thing in existing code in this package.

I *very strongly* suggest that you pick a different argument name for the two naming conventions, to avoid this confusion.

I suggest:
* `component`: the SAL component name, e.g. "Hexapod:1"
 * `component_attr`: the component attribute name in `RemoteGroup`: e.g. "hexapod_1".

salobj.SalInfo has something a bit similar for topic names.

Reply:
Loved this idea! thanks!